README.md

```diff
@@ -188,7 +188,7 @@ MaxDiffusion started as a fork of [Diffusers](https://github.com/huggingface/dif
 
 Whether you are forking MaxDiffusion for your own needs or intending to contribute back to the community, a full suite of tests can be found in `tests` and `src/maxdiffusion/tests`.
 
-To run unit tests, simply run:
+To run unit tests, you'll need to [install gcsfuse](https://cloud.google.com/storage/docs/cloud-storage-fuse/install) then simply run:
```

Comment:
Do we recommend users to run setup.sh? It has gcsfuse installation: https://github.com/AI-Hypercomputer/maxdiffusion/blob/main/setup.sh#L28-L32. But I don't see it mentioned in README.md

Reply:
We already use it for github unit test: https://github.com/AI-Hypercomputer/maxdiffusion/blob/main/.github/workflows/UnitTests.yml#L43

Reply:
@aireenmei please take another look.